ext/PlottingExt/ViewCrystal.jl

```diff
@@ -60,6 +60,50 @@ function propagate_reference_bond_for_cell(cryst, b_ref)
     return reduce(vcat, found)
 end
 
+# Get the quadratic anisotropy as a 3×3 exchange matrix for atom `i` in the
+# chemical cell.
+function quadratic_anisotropy(sys, i)
```

Comment:
I wonder if maybe an asterisk or warning should be given (in the display) if c4 or c6 is non-zero?

My worry is just that one of the selling points of Sunny has been its support for completely general anisotropies. This potentially truncated representation of anisotropies might be misleading to someone who wants to use the advanced anisotropy features and specifically looks at the viewer to check things.

Reply:
Good idea. Now `view_crystal(sys)` will display the text _"+ higher order terms"_ if there are any terms beyond the quadratic level. This behavior has been added for both pair couplings (on bonds) and anisotropy (on sites).